extensions/shared/components/upgrade-nudge/index.jsx

```diff
@@ -32,6 +32,21 @@ const getTitle = ( customTitle, planName ) => {
 		: __( 'Upgrade to a paid plan to use this block on your site.', 'jetpack' );
 };
 
+const getSubtitle = ( subtitle ) => {
+	if ( subtitle === false ) {
+		return null;
+	}
```

Comment:
Why do we have an explicit check for `false`?

Reply:
When the given subtitle value of the Nudge is null or undefined, it shows a default subtitle. But when it's False, it doesn't show it.